presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -2298,6 +2298,12 @@ public void testJoinWithInSubqueryToBeExecutedAsPostJoinFilter()
         }
     }
 
+    @Test
+    public void testAggregationPushdownThroughOuterJoinNotFiringInCorrelatedAggregates()
+    {
+        assertQuery("select x, max(x) from (select * from (values (1)) t(x) left join (values (1)) t2(y) on t.x = t2.y) group by x");
```

Comment:
- was this query failing before? If so please open an issue with a problem that you are solving here
 - please use capital letters for SQL keywords

Reply:
`select * from (values (1))` can be rewritten as just `values 1`